terraform/lambda.tf

```
@@ -0,0 +1,48 @@
+resource "aws_lambda_function" "probot_handler" {
+  depends_on    = [aws_cloudwatch_log_group.probot_handler]
+  filename      = "dist/probot-${var.deployment_version}.zip"
```

```suggestion
  filename      = "dist/probot.zip"
```

A version number is not needed here.